app/models/tag.rb

```diff
@@ -215,4 +215,8 @@ def self.trending(limit = 5 , start_date = DateTime.now - 1.month , end_date = D
        .order('count DESC')
        .limit(limit)
   end
+
+  def self.find_id_of_everything_tag
```

Comment:
Also here, I guess I'd just like to suggest using `Tag.find_by_name('everything').tid` instead of making a new one-line model method. I just am not sure we'll be using it again, so why add 3 more lines, you know? I do really like how you're thinking in terms of code modularity though!